### DISCUSSION TITLE
Looks awesome!

### DISCUSSION THREAD

**Discussion by MephistoLa** [MuertoGB/MacEfiToolkit#16] [Q&A] [answered]
... but how should I run it? Ive an efi locked Mac that neeeeds me to figure this out. 

Ayuda me por favor!

**Comment by MuertoGB** [comment, marked as answer]
Please see the manual, section "EFI Lock".:
https://github.com/MuertoGB/MacEfiToolkit/blob/main/MANUAL.md#efi-lock

Dump the SPIROM firmware, drag the dump into Mac EFI Toolkit, click to unlock, agree to terms, Mac EFI Toolkit will provide you an unlocked firmware.
